dotcom-rendering/src/web/components/OnwardsUpper.importable.tsx

```diff
@@ -47,16 +47,11 @@ const ALLOWED_TAGS = [
 	'football/liverpool',
 ];
 
+/** This function looks for the first tag in pageTags, that also exists in our allowlist */
```

Comment:
This method is actually not doing what one might expect: https://github.com/guardian/dotcom-rendering/pull/1078#issuecomment-1446489263

Reply:
Fix in https://github.com/guardian/dotcom-rendering/pull/7300